src/trackedit/internal/au3/au3interaction.cpp

```diff
@@ -892,18 +931,22 @@ bool Au3Interaction::removeTracksData(const TrackIdList& tracksIds, secs_t begin
     return true;
 }
 
-bool Au3Interaction::moveClips(secs_t offset, bool completed)
+bool Au3Interaction::moveClips(secs_t horizontalOffset, int verticalOffset, bool completed)
```

Comment:
horizontalOffset and verticalOffset should have same type

Reply:
there is no need to use the term "horizontal" here. At this level we know nothing about the representation...
It is probably something like: `timeOffset`, `timePositionOffset`

see also: https://github.com/audacity/audacity/pull/8031#discussion_r1914594270